test/services/helpers/services/roles.js

```diff
@@ -14,11 +19,11 @@ const cleanup = () => {
 	}
 	const ids = createdRoles;
 	createdRoles = [];
-	return Role.deleteMany({ id: { $in: ids } });
+	return RoleModel.deleteMany({ _id: { $in: ids } }).lean().exec();
 };
 
-module.exports = {
-	create,
+module.exports = (app, opt) => ({
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['opt' is defined but never used. Allowed unused args must match /app|req|res|next|options|params/u.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5258109)